lib/config/recommended.js

```diff
@@ -16,4 +17,14 @@ module.exports = {
   plugins: ['ember'],
 
   rules,
+
+  overrides: [
+    {
+      files: ['**/*.gjs', '**/*.gts'],
+      processor: 'ember/<template>',
+      globals: {
+        [util.TEMPLATE_TAG_PLACEHOLDER]: 'readonly',
```

Comment:
Can you add a comment explaining what this global is for?

Reply:
Included in the comment about the override. thanks!